packages/react-components/react-field/src/components/Field/useField.tsx

```diff
@@ -97,13 +90,11 @@ export const useField_unstable = (props: FieldProps, ref: React.Ref<HTMLDivEleme
     components: {
       root: 'div',
       label: Label,
-      validationMessage: 'div',
-      validationMessageIcon: 'span',
-      hint: 'div',
+      validationMessage: FieldMessage,
+      hint: FieldMessage,
```

Comment:
Does having the hint as a FieldMessage add value over div?  Could the component type for hint be div or FieldMessage?

Reply:
The main reason was because both messages share much of the same styling. A secondary reason is to allow an icon slot for the hint message; but that is not a design requirement. I agree with @smhigley that it's weird to have validationState on the hint message, although it isn't exposed at the top level of Field.

If hint weren't a FieldMessage, then it might be better to not add the FieldMessage component, and leave both as divs styled by the Field itself.